sonata-project/page-bundle/4.0/config/packages/sonata_page.yaml

```diff
@@ -0,0 +1,80 @@
+sonata_block:
+    default_contexts: [sonata_page_bundle]
+
+sonata_admin:
+    assets:
+        extra_javascripts:
+            - bundles/sonatapage/app.js
+        extra_stylesheets:
+            - bundles/sonatapage/app.css
+
+sonata_page:
+    multisite: host
+    use_streamed_response: false
+
+    router_auto_register:
+        enabled: true
+        priority: 150
+    ignore_route_patterns:
+        - ^(.*)admin(.*)   # ignore admin route, ie route containing 'admin'
+        - ^_(.*)          # ignore symfony routes
+    ignore_routes:
+        - sonata_page_cache_esi
+        - sonata_page_cache_ssi
+        - sonata_page_js_sync_cache
+        - sonata_page_js_async_cache
+        - sonata_cache_esi
+        - sonata_cache_ssi
+        - sonata_cache_js_async
+        - sonata_cache_js_sync
+        - sonata_cache_apc
+    ignore_uri_patterns:
+        - ^/admin\/   # ignore admin route, ie route containing 'admin'
+
+    default_template: default
+    templates:
+        default:
+            path: '@SonataPage/layout.html.twig'
+            name: 'default'
+            containers:
+                header:
+                    name: Header
+                content_top:
+                    name: Top content
+                content:
+                    name: Main content
+                content_bottom:
+                    name: Bottom content
+                footer:
+                    name: Footer
+            matrix:
+                layout: |
+                    HHHHHHHH
+                    HHHHHHHH
+                    TTTTTTTT
+                    TTTTTTTT
+                    CCCCCCCC
+                    CCCCCCCC
+                    BBBBBBBB
+                    BBBBBBBB
+                    FFFFFFFF
+                    FFFFFFFF
+
+                mapping:
+                    H: header
+                    T: content_top
+                    C: content
+                    B: content_bottom
+                    F: footer
+
+    direct_publication: '%kernel.debug%'
+
+    catch_exceptions:
+        not_found: [404]    # render 404 page with "not_found" key (name generated: _page_internal_error_{key})
+        fatal:     [500]    # so you can use the same page for different http errors or specify specific page for each error
+
+    class:
+        page: 'App\Entity\SonataPagePage'
+        snapshot: 'App\Entity\SonataPageSnapshot'
+        block: 'App\Entity\SonataPageBlock'
+        site: 'App\Entity\SonataPageSite'
```

Comment:
I dunno if we're supposed to add this in the recipe:
- It force using DoctrineORM
- It force the naming of the entities
- It used annotation on entities when people might prefer attributes
- It arbitrary added index on some entities

I dunno how similar bundle/recipe works for these situations.

Reply:
It's not "necessary". I mean people can use another entity instead.
I'm not familiar about recipe and what should be provided or not.
Dunno if we can find a similar library recipe...

Reply:
should be good the recipe help the user make the sonata page bundle works without do too much!

I know it's used for ORM, But ATM I don't have any solution to make this works for both, and if I comment or remove this configuration, it will be even worst, because it will broken to ORM and ODM, then IMO we could let this like this and in the feature we can provide a new solution!